### PR TITLE
Add register_web_login method in order to support session_token requests

### PR DESCRIPTION
**Pull request by brunoekferrari**
In order to avoid breaking existing integrations, I created a new method based on the existing register_login in order to support session_token requests using the Python lib.

### CODE REVIEW

**Review by gaabs — COMMENTED** [review]
Please, add tests related to this method on https://github.com/inloco/incognia-python/blob/main/tests/test_api.py :pray: 

Also, it would be very nice to add support to Web Signup as well :eyes:

**Comment by brunoekferrari** [comment]
> Please, add tests related to this method on https://github.com/inloco/incognia-python/blob/main/tests/test_api.py 🙏
> 
> Also, it would be very nice to add support to Web Signup as well 👀

Sure, I will add them all. I guess web signup is not available yet, so there's no way to test it. Shall I update it anyway or wait?

File: tests/test_api.py [hunk 1]
```diff
@@ -376,3 +388,67 @@ def test_register_login_when_required_fields_are_invalid_should_raise_an_Incogni
...
+
+        api = IncogniaAPI(self.CLIENT_ID, self.CLIENT_SECRET)
+
+        request_response = api.register_web_login(self.SESSION_TOKEN, self.ACCOUNT_ID)
+
+        mock_token_manager_get.assert_called()
+        mock_base_request_post.assert_called_with(Endpoints.TRANSACTIONS,
+                                                  headers=self.AUTH_AND_JSON_CONTENT_HEADERS,
+                                                  params=self.DEFAULT_PARAMS,
+                                                  data=self.REGISTER_VALID_LOGIN_DATA)
```
Comment (b0rba) [inline tests/test_api.py:407]: Use `REGISTER_VALID_WEB_LOGIN_DATA`

File: tests/test_api.py [hunk 2]
```diff
@@ -376,3 +388,67 @@ def test_register_login_when_required_fields_are_invalid_should_raise_an_Incogni
...
+
+        self.assertRaises(IncogniaHTTPError, api.register_web_login,
+                          session_token=self.INVALID_SESSION_TOKEN,
+                          account_id=self.INVALID_ACCOUNT_ID)
+
+        mock_token_manager_get.assert_called()
+        mock_base_request_post.assert_called_with(Endpoints.TRANSACTIONS,
+                                                  headers=self.AUTH_AND_JSON_CONTENT_HEADERS,
+                                                  params=self.DEFAULT_PARAMS,
+                                                  data=self.REGISTER_INVALID_LOGIN_DATA)
```
Comment (b0rba) [inline tests/test_api.py:454]: Use REGISTER_INVALID_WEB_LOGIN_DATA

**Comment by b0rba** [comment]
#44

**Pull request closed without merging**